code/modules/admin/verbs/smite.dm

```diff
@@ -0,0 +1,83 @@
+/client/proc/smite(var/mob/living/carbon/human/target in player_list)
+	set name = "Smite"
+	set desc = "Abuse a player with various 'special treatments' from a list."
+	set category = "Fun"
+	if(!check_rights(R_FUN))
+		return
+
+	if(!istype(target))
+		return
+
+	var/list/smite_types = list(SMITE_BREAKLEGS,SMITE_BLUESPACEARTILLERY,SMITE_SPONTANEOUSCOMBUSTION,SMITE_LIGHTNINGBOLT)
+
+	var/smite_choice = input("Select the type of SMITE for [target]","SMITE Type Choice") as null|anything in smite_types
+	if(!smite_choice)
+		return
+
+	switch(smite_choice)
+		if(SMITE_BREAKLEGS)
+			var/broken_legs = 0
+			var/obj/item/organ/external/left_leg = target.get_organ(BP_L_LEG)
+			if(left_leg && left_leg.fracture())
+				broken_legs++
+			var/obj/item/organ/external/right_leg = target.get_organ(BP_R_LEG)
+			if(right_leg && right_leg.fracture())
+				broken_legs++
+			if(!broken_legs)
+				to_chat(src,"[target] didn't have any breakable legs, sorry.")
+		
+		if(SMITE_BLUESPACEARTILLERY)
+			bluespace_artillery(target,src)
+		
+		if(SMITE_SPONTANEOUSCOMBUSTION)
+			target.adjust_fire_stacks(10)
+			target.IgniteMob()
+			target.visible_message("<span class='danger'>[target] bursts into flames!</span>")
+		
+		if(SMITE_LIGHTNINGBOLT)
+			var/turf/T = get_step(get_step(target, NORTH), NORTH)
+			T.Beam(target, icon_state="lightning[rand(1,12)]", time = 5)
+			target.adjustFireLoss(75)
```

Comment:
Could this not be done with electrocute_act() ?

Reply:
It'd only shock hands, and be resisted by gloves.

Reply:
electrocute_act can be passed a bodypart to target, can it not?

Reply:
I guess, but then what? I just pick() from the list of all their body parts and randomly only their right leg gets burned? :P I dunno, the overall seems like a nice smitey touch. Probably won't sever any limbs since all spread out.

Reply:
Was thinking head or chest. Might not sever/ash a limb, but that's I think still 75 burn and 10 seconds of weaken, and burns that bad are fairly nasty.

Reply:
Not really a big deal, but it feels more thematic, and if I really want to screw someone over for praying something silly, I have more dangerous methods.

Reply:
Did it!